src/icon/Icon.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export type IconProps = {
+    iconName: string;
+    className?: string;
+};
+
+export default function Icon(props: IconProps) {
+    return <span className={`en-icon en-icon-${props.iconName} ${props.className}`} aria-hidden="true" />
```

Comment:
```suggestion
    return <span className={`en-icon en-icon-${props.iconName} ${props.className ?? ''}`} aria-hidden="true" />
```

Since this is optional you need to pass it a fallback. Spotted this in the snapshot below :trollface: